gobblin-core/src/main/java/gobblin/source/extractor/extract/jdbc/JdbcExtractor.java

```diff
@@ -317,11 +317,13 @@ public void extractMetadata(String schema, String entity, WorkUnit workUnit) thr
           JsonObject jsonObject = gson.fromJson(jsonStr, JsonObject.class).getAsJsonObject();
           targetSchema.add(jsonObject);
           headerColumns.add(targetColumnName);
+          sourceColumnName = "`" + sourceColumnName + "`";
```

Comment:
Done.
SQLServer supports it when quoted_identifier option is set on. MySQL supports it when ANSI mode is on. All other systems support double quotes by default.
Default for SQLServer is [ ] , default for MySQL is backtick.